include/aikido/planner/ConfigurationToEndEffectorOffset.hpp

```diff
@@ -0,0 +1,85 @@
+#ifndef AIKIDO_PLANNER_PLANTOENDEFFECTOROFFSET_HPP_
+#define AIKIDO_PLANNER_PLANTOENDEFFECTOROFFSET_HPP_
+
+#include <dart/dart.hpp>
+#include "aikido/constraint/Testable.hpp"
+#include "aikido/planner/Problem.hpp"
+#include "aikido/statespace/StateSpace.hpp"
+#include "aikido/trajectory/Interpolated.hpp"
+
+namespace aikido {
+namespace planner {
+
+/// Planning problem to plan to desired en-effector offset with fixed
+/// orientation
+class ConfigurationToEndEffectorOffset : public Problem
+{
+public:
+  /// Constructor.
+  ///
+  /// \param stateSpace State space.
+  /// \param bodyNode Body Node or robot for which the path is to be planned.
```

Comment:
Is this code written this way because we think it'd be valuable for a robot (e.g. RACECAR) that doesn't have an end-effector to be able to plan to an offset in its current direction?

Reply:
Maybe change the name of the problem also in that case? End Effector is very much manipulator specific. We'd like the planners and problems to be generic no? Maybe `SingleSourceSingleTargetFixedOrientationProblem`. Took me a while to type that name. :P [Still not a good enough name though]

Reply:
I think this is just about confusing description. I renamed it to `endEffectorBodyNode` (and getters and setters accordingly) and improved the descriptions.